src/client/App.jsx

```diff
@@ -8,9 +8,9 @@ import { Competition } from './components/Competition';
 import { Season } from './components/Season';
 import { CreateSeason } from './components/CreateSeason';
 
+import Competitors from './components/Competitors';
```

Comment:
3. This needs changing to

<code>import  { Competitors } from './components/Competitors';</code>